CHANGELOG_UNRELEASED.md

```diff
@@ -13,6 +13,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Added a `void` notation for the `Empty_set` type of the standard library, the
   canonical injection `of_void` and its cancellation lemma `of_voidK`, and
   `eq`, `choice`, `count` and `fin` instances.
+  
+- Added `ltn_ind` general induction principle for `nat` variables, helper lemmas `ubnP` and `ltnSE` in support of a generalized induction idiom for `nat` mesures that does not rely on the `{-2}` numerical occurrence selector, and purged this idiom from the `mathcomp` library. The `{-2}` idiom is likely to fail in upcoming releases for cardinal (i.e., `#|A|`) measures as the new `#|A|` notation may hide multiple occurrences of `A`.
```

Comment:
mesures --> measures

Reply:
Well spotted; should I also include the more direct call for clients to update as well (which I added to the PR description), perhaps splitting the paragraph between the `Added` and `Changed` sections?

Reply:
I agree adding the direct call for clients to update is a good use of the CHANGELOG file.

Reply:
@ggonthier I propose to merge as soon as you implement @ybertot 's suggestion.

Reply:
Done (and squashed), along with the inclusion of the additional helpers that arose from the conversation with @CohenCyril.